packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -445,13 +427,13 @@ class DDSTableOfContents extends StableSelectorMixin(LitElement) {
   connectedCallback() {
     super.connectedCallback();
     this._cleanAndCreateObserverResizeMobileContainer({ create: true });
-    this._cleanAndCreateObserverIntersectionTarget({ create: true });
+    this._cleanAndCreateScrollListener({ create: true });
```

Comment:
Same as previous `lodash` comment, can use [`throttle`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/back-to-top/back-to-top.ts#L119-L125) here.